discovery/README.md

```diff
@@ -2,50 +2,18 @@
 
 The Discovery agent finds deployed API Proxies in Apigee then sends them to API Central
 
-## Build and run
+## Executing
 
-The following make targets are available
-
-| Target          | Description                                                    | Output(s)                     |
-| --------------- | -------------------------------------------------------------- | ----------------------------- |
-| dep             | downloads all dependencies needed to build the discovery agent | /vendor                       |
-| test            | runs go test against all test files int he repo                | test results                  |
-| update-sdk      | pulls the latest changes to main on the SDK repo               |                               |
-| build           | builds the binary discovery agent                              | bin/apigee_discovery_agent    |
-| apigee-generate | generates the models for the Apigee APIs                       | pkg/apigee/models             |
-| docker-build    | builds the discovery agent in a docker container               | apigee-discovery-agent:latest |
-
-### Build (Docker)
+The Apigee Discovery Agent is built and distributed as a docker image. To execute the agent with the image the following command may be executed, updating the version.
```

Comment:
the second sentence seems klunky to me...

"To execute the agent using the image, the following command may be executed.  The result is an updated version."

something like that?

Reply:
"The Apigee Discovery Agent is built and distributed as a docker image. To execute the agent using the image, the following command may be executed. Update the version in the examples below."

Reply:
i like it...